test/Microsoft.AspNetCore.Mvc.Razor.Test/Internal/CSharpCompilerTest.cs

```diff
@@ -151,7 +151,7 @@ public void Constructor_ConfiguresLanguageVersion()
             // Arrange
             var dependencyContextOptions = new DependencyContextCompilationOptions(
                 new[] { "MyDefine" },
-                languageVersion: "CSharp7_1",
+                languageVersion: "7.1",
```

Comment:
I could add more scenarios, but at that point, we'd be testing Roslyn's code base.